client/systems.go

```diff
@@ -155,10 +156,21 @@ const (
 	StorageEncryptionSupportDefective = "defective"
 )
 
+type StorageEncryptionFeature string
+
+const (
+	// Indicates that passphrase authentication is available.
+	StorageEncryptionFeaturePassphraseAuth StorageEncryptionFeature = "passphrase-auth"
+	// TODO: Add PIN option when secboot support lands.
```

Comment:
you can probably at the value for it right now, it'll be rejected at later stage as not supported, won't it?